### ISSUE TITLE
attach mode doesn't work on v0.2.0

### ISSUE THREAD

**Issue by Kounotori-ssbm** [ruby/vscode-rdbg#250]
Hello, thank you a lot for this awesome extension!

I think vscode debugging for attach mode had worked until yesterday or the day before yesterday.
But after releasing v0.2.0 (maybe), when I start debugging this dialog appears and failed to start debugging.
![image](https://github.com/ruby/vscode-rdbg/assets/19996147/594a6ef2-e4ae-4e78-a068-0b3a55f2a328)
```
bash: cannot set terminal process group (173): Inappropriate ioctl for device
bash: no job control in this shell
```

And, output tab for rdbg says same messages.
![image](https://github.com/ruby/vscode-rdbg/assets/19996147/7601ad33-b673-40bf-a02b-3a5359ecdcae)

## My environment
- Ruby = v3.2.0
- Rails = v7.0.4.2
- rdbg = v.1.7.1
- docker-compose (Docker = v20.10.11, docker-compose = v1.29.2)
- Dev Containers extension v0,292.0
- Windows WSL2 (Ubuntu 20.04)

`docker-compose.yml`
```yml
services:
  api 
    # ...
    command: /bin/sh -c "rm -f tmp/pids/server.pid && rdbg --nonstop --open --command -- rails server --binding=0.0.0.0"
```

`launch.json`
```json
{
  "version": "0.2.0",
  "configurations": [
    {
      "type": "rdbg",
      "name": "Attach with rdbg",
      "request": "attach"
    }
  ]
}
```

## v0.1.0 works
After reinstalling v0.1.0, debugging for attach mode has worked again.
![image](https://github.com/ruby/vscode-rdbg/assets/19996147/c8744ead-ae23-4ccb-ac3a-1c80871fab98)

Do I overlook something to use v0.2.0 for attach mode?

**Comment by andyw8** [comment]
Are you using a version manager? I think this may be due to that not being set:

https://github.com/ruby/vscode-rdbg/#selecting-a-version-manager

**Comment by stage-rl** [comment]
Same problem here on Debian. No version manager in use, and the extension setting is correct (None - Default).

**Comment by ono-max** [comment]
Thank you for reporting. 
Could you show us the sample program to reproduce this problem?

**Comment by stage-rl** [comment]
E.g. this repo:
https://github.com/stage-rl/rails_admin

**Comment by stage-rl** [comment]
Fixed locally, pls replace '-lic' by '-c' when invoking bash (my understanding is you don't want interactive shell). Haven't tested with other shells, so definately test, before merging. Thanks

**Comment by ono-max** [comment]
Thank you for your corporation. I think that https://github.com/ruby/vscode-rdbg/pull/259 fixes it.

**Comment by ono-max** [comment]
The new version has been released. If you still have a problem, please mention me or create the issue. Thank you.

**Comment by ono-max** [comment]
@plfoley 

Sorry, we need more information. Could you show us the sample program to reproduce this problem?

**Comment by plfoley** [comment]
I am attaching to a Rails instance running in a separate terminal.
Rails is launched using the following command:
`bundle exec ~/.rbenv/shims/rdbg --open -n -c -- bundle exec rails s`
Just to be extra clear: I am not running a single Ruby file here.

This works perfectly fine in version 0.1.0 and stopped working in version 0.2.0
I also tried to update the debug gem to the latest version with no effect.

It is possible to attach using a port in version 0.2.1 but when I do this, my breakpoints don't get triggered. Breaking on exception works in this context tough.

Last useful piece of information I can think of: I am using zsh.

If you need more information of help diagnose the issue, please feel free to ask.
This is not a blocking issue as reverting to version 0.1.0 allows me to do what I need, but this tool has been really helpful so far I'd love to see this fixed.

**Comment by ono-max** [comment]
@plfoley 

Sorry, I'm a little bit confused. Do you get the following messages?

```
bash: cannot set terminal process group (173): Inappropriate ioctl for device
bash: no job control in this **shell**
```

If not, can you create a new issue? This issue is for fixing the bash issue.
Thank you.

**Comment by plfoley** [comment]
@ono-max So sorry I didn't realized the issue was related to bash only.
I changed my login shell to bash and the issue is fixed in version 0.2.1 when using bash as the login shell on macOS.

There is an issue with zsh on macOS however. I will create a new issue for this.
